release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1011,6 +1024,54 @@ KMM includes the following features for connected environments only:
 ==== Hub and spoke cluster support (Technology Preview)
 
 For hub and spoke deployments in an environment that can access the internet, you can use the kernel module management (KMM) Operator deployed in the hub cluster to manage the deployment of the required kernel modules to one or more managed clusters. 
+[id="scalability-and-performance-talm-updates"]
+==== {cgu-operator-first}
+
+{cgu-operator-first} now provides more detailed status information and messages, and redesigned conditions. 
+You can use the `ClusterLabelSelector` field for greater flexibility in selecting clusters for update.
+You can use timeout settings to determine what happens if an update fails for a cluster, for example, skipping the failing cluster and continuing to upgrade other clusters, or stopping policy remediation for all clusters. 
+For more information see xref:../scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc[Topology Aware Lifecycle Manager for cluster updates].
+
+[id="scalability-and-performance-mount-namespace"]
+==== Mount namespace encapsulation (Technology Preview)
+
+Encapsulation is the process of moving all Kubernetes-specific mount points to an alternative namespace to reduce the visibility and performance impact of a large number of mount points in the default namespace. Previously, mount namespace encapsulation has been deployed transparently in {product-title} specifically for Distributed Units (DUs) installed using GitOps ZTP. In {product-title} v4.12, this proven functionality is now available as a configurable option.
```

Comment:
s/{product-title}/{product-title},

I might remove _proven_. It sounds to me like marketing-speak.

Reply:
True. I've removed it.